controllers/etcd_custodian_controller.go

```diff
@@ -108,6 +122,37 @@ func (ec *EtcdCustodian) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.
 		return ctrl.Result{}, err
 	}
 
+	conLength := len(etcd.Status.Conditions)
+	if conLength > 0 && etcd.Status.Conditions[conLength-1].Reason == "QuorumLost" && etcd.Spec.Replicas > 1 {
+		logger.Info("Quorum loss detected. Taking measures to fix it.")
+		if !ec.config.EnableAutomaticQuorumLossHandling {
```

Comment:
Why would we want to update ETCD status if the quorum is lost? If quorum is lost and ETCD status is updated, ETCD controller may start working to fix the ETCD . This will give totally unintended result

Reply:
Ah, okay
Fair enough